airflow/www/app.py

```diff
@@ -66,8 +66,19 @@ def create_app(config=None, testing=False, app_name="Airflow"):
     flask_app = Flask(__name__)
     flask_app.secret_key = conf.get('webserver', 'SECRET_KEY')
 
-    session_lifetime_days = conf.getint('webserver', 'SESSION_LIFETIME_DAYS', fallback=30)
-    flask_app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(days=session_lifetime_days)
+    default_session_lifetime_minutes = 43200
+    if conf.has_option('webserver', 'SESSION_LIFETIME_MINUTES'):
+        session_lifetime_minutes = conf.getint('webserver', 'SESSION_LIFETIME_MINUTES',
+                                               fallback=default_session_lifetime_minutes)
+    else:
+        warnings.warn('`SESSION_LIFETIME_DAYS` option has been renamed to `SESSION_LIFETIME_MINUTES`. '
```

Comment:
Can you also add the section name to this warning message?

Reply:
done